btc-lsp/src/BtcLsp/Rpc/ScriptHashRpc.hs

```diff
@@ -0,0 +1,139 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module BtcLsp.Rpc.ScriptHashRpc
+  ( getBalance,
+    version,
+    Balance (..),
+    Address (..),
+    ScriptHash (..),
+  )
+where
+
+import BtcLsp.Import
+import BtcLsp.Rpc.Client as Client
+import BtcLsp.Rpc.RpcRequest as Req
+import BtcLsp.Rpc.RpcResponse as Resp
+import Data.Aeson (decode, encode, object, withObject, (.:), (.=))
+import qualified Data.ByteString.Lazy as BS
+import Data.Digest.Pure.SHA
+import Network.HTTP.Client as HTTPClient hiding (Proxy)
+import qualified Text.Hex as TH
+
+newtype Address = Address Text
```

Comment:
There is already existing `OnChainAddress` type, let's reuse it

Reply:
Good idea